models/user.go

```diff
@@ -526,6 +542,16 @@ func (u *User) IsUserOrgOwner(orgID int64) bool {
 	return isOwner
 }
 
+// IsUserPartOfOrg returns true if user is part of the organization
+func (u *User) IsUserPartOfOrg(userID int64) bool {
```

Comment:
Hmm. I'm confused by this function signature & comment.

I guess that u is meant to be the organisation, and you're checking that user ID is in this organisation. 

I'm not sure what the name of this function should be to completely ensure it's clear but you should definitely change the comment saying: returns true is user ID is part of the u organisation.

I also have to admit that I have a little pet hate for functions that hide database access like this - it's too easy to miss in a transaction.

Reply:
I tried to be clever and change u to org, however lint (rightfully so), error'd out. So I've updated comment to match your suggestion.

Reply:
Thanks!

Reply:
Oh change it to the `user with id userID is part of the organization, u.` or `userID is part of the organization`. It's still slightly unclear if you're just grepping the source code.

Reply:
oops. Thanks for checking again after the first update. Hopefully I got it right this time.

Reply:
Cools! Thanks dude! I've just re-hit the extremely confusing similarly named functions:

https://github.com/go-gitea/gitea/blob/master/models/ssh_key.go#L765

&

https://github.com/go-gitea/gitea/blob/master/models/ssh_key.go#L777